uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -59,57 +54,51 @@ function tasksassignments_civicrm_install() {
 }
 
 /**
- * Implementation of hook_civicrm_uninstall
+ * Implements hook_civicrm_uninstall().
  *
  * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_uninstall
  */
 function tasksassignments_civicrm_uninstall() {
   // PCHR-1263 : hrcase should not be installed/enabled without Task & Assignments extension
-  if (tasksassignments_checkHrcase())  {
+  if (tasksassignments_checkHrcase()) {
     tasksassignments_extensionsPageRedirect();
   }
   _tasksassignments_civix_civicrm_uninstall();
 }
 
 /**
- * Implementation of hook_civicrm_enable
+ * Implements hook_civicrm_enable().
  *
  * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_enable
  */
 function tasksassignments_civicrm_enable() {
   _tasksassignments_toggleMenuItems();
-  _tasksassignments_toggleCaseMenuItems(false);
+  _tasksassignments_toggleCaseMenuItems(FALSE);
 
   CRM_Core_BAO_Navigation::resetNavigation();
   _tasksassignments_civix_civicrm_enable();
 }
 
 /**
- * Implementation of hook_civicrm_disable
+ * Implements hook_civicrm_disable().
  *
  * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_disable
  */
 function tasksassignments_civicrm_disable() {
   // PCHR-1263 : hrcase should not be installed/enabled without Task & Assignments extension
-  if (tasksassignments_checkHrcase())  {
+  if (tasksassignments_checkHrcase()) {
     tasksassignments_extensionsPageRedirect();
   }
 
-  _tasksassignments_toggleMenuItems(false);
+  _tasksassignments_toggleMenuItems(FALSE);
   _tasksassignments_toggleCaseMenuItems();
 
   CRM_Core_BAO_Navigation::resetNavigation();
   _tasksassignments_civix_civicrm_disable();
 }
 
 /**
- * Implementation of hook_civicrm_upgrade
- *
- * @param $op string, the type of operation being performed; 'check' or 'enqueue'
- * @param $queue CRM_Queue_Queue, (for 'enqueue') the modifiable list of pending up upgrade tasks
- *
- * @return mixed  based on op. for 'check', returns array(boolean) (TRUE if upgrades are pending)
- *                for 'enqueue', returns void
+ * Implements hook_civicrm_upgrade().
```

Comment:
Please add the parameter doc block back.

Did you make this changes manually or you used a code sniffer to correct automatically?